cmd/adapter/adapter.go

```diff
@@ -90,6 +92,10 @@ func (cmd *PrometheusAdapter) makePromClient() (prom.Client, error) {
 		return nil, fmt.Errorf("invalid Prometheus URL %q: %v", baseURL, err)
 	}
 
+	if cmd.PrometheusVerb != http.MethodGet && cmd.PrometheusVerb != http.MethodPost {
+		return nil, fmt.Errorf("unsupported Prometheus Http Verb %q", cmd.PrometheusVerb)
```

Comment:
```suggestion
		return nil, fmt.Errorf("unsupported Prometheus HTTP verb %q", cmd.PrometheusVerb)
```
Could you also add the list of possible verb value to this error to make it simpler for the users?